packages/server/src/common/utils/console-logger.ts

```diff
@@ -33,48 +33,50 @@ export class ConsoleLogger extends Logger {
 
     info(message: string, ...params: any[]): void {
         if (LogLevel.info <= this.logLevel) {
-            console.info(this.logMessage(message, params));
+            console.info(this.logMessage(LogLevel.info, message, params));
         }
     }
 
     warn(message: string, ...params: any[]): void {
         if (LogLevel.warn <= this.logLevel) {
-            console.warn(this.logMessage(message, params));
+            console.warn(this.logMessage(LogLevel.warn, message, params));
         }
     }
 
     error(message: string, ...params: any[]): void {
         if (LogLevel.error <= this.logLevel) {
-            console.error(this.logMessage(message, params));
+            console.error(this.logMessage(LogLevel.error, message, params));
         }
     }
 
     debug(message: string, ...params: any[]): void {
         if (LogLevel.debug <= this.logLevel) {
-            console.debug(this.logMessage(message, params));
+            console.debug(this.logMessage(LogLevel.debug, message, params));
         }
     }
 
-    logMessage(message: string, ...params: any[]): string {
-        const timestamp = new Date().toLocaleTimeString();
+    logMessage(level: LogLevel, message: string, ...params: any[]): string {
         const caller = this.caller ? `[${this.caller}]` : '';
         const additional = this.logAdditionals(...params);
-        return `${timestamp} ${this.logLevel} - ${caller} ${message} ${additional}`;
+        return `${this.timestamp()} ${asLogLevel(level)} - ${caller} ${message} ${additional}`;
+    }
+
+    protected timestamp(): string {
+        return new Date().toLocaleTimeString();
     }
 
     logAdditionals(...params: any[]): string {
-        if (!params || params.length === 0) {
-            return '';
-        }
-        return params.map(param => this.stringify(param)).join(',\n');
+        const extractedParams = params.length === 1 && Array.isArray(params[0]) ? params[0] : params;
```

Comment:
Could you explain the reasoning behind this change?
I don't see why its necessary